lib/reek/cli/command/todo_list_command.rb

```diff
@@ -44,6 +46,14 @@ def groups
               todos.inject(&:merge)
             end
         end
+
+        def write_to_file
+          File.open(DEFAULT_CONFIGURATION_FILE_NAME, 'w') do |configuration_file|
+            configuration_file.write HEADER
+            configuration_file.write({ DETECTOR_KEY => groups }.to_yaml)
+          end
+          puts "\n'#{DEFAULT_CONFIGURATION_FILE_NAME}' generated! You can now use this as a starting point."
```

Comment:
I'd move the puts out of `#write_to_file`, so all output is concentrated in `#execute`.